### PR TITLE
feat(SmallToggle): Add small toggle component

### PR DESCRIPTION
**Pull request by alisonjoseph**
https://github.ibm.com/Bluemix/carbon-issues/issues/453

#### Changelog

**New**

Small Toggle Component

### CODE REVIEW

File: src/components/ToggleSmall/ToggleSmall.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <path d="M2.2403 2.7299L4.9245 0 6 1.1117 2.2384 5 0 2.6863 1.0612 1.511z" />
+          </svg>
+        </span>
+      </label>
+    </div>
+  );
+};
+
+ToggleSmall.propTypes = {
+  className: PropTypes.string,
```
Comment (asudoh) [inline src/components/ToggleSmall/ToggleSmall.js]: Great to have comments explaining what those properties are for. If you do that, `Info` button at the top right will show those properties. The comment should be styled in:

```javascript
/**
 * `true` to make it toggled by default.
 */
defaultToggled: PropTypes.bool,
```

File: src/components/ToggleSmall/ToggleSmall.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  defaultToggled: PropTypes.bool,
+  onToggle: PropTypes.func,
+  id: PropTypes.string.isRequired,
+  toggled: PropTypes.bool,
+  small: PropTypes.bool,
```
Comment (asudoh) [inline src/components/ToggleSmall/ToggleSmall.js]: Did you mean to remove this prop (along with the `defaultProps`)?

Comment (alisonjoseph) [reply src/components/ToggleSmall/ToggleSmall.js]: Oops, I don't need the small prop. Is that what you are referring to, or something else?

File: src/components/ToggleSmall/ToggleSmall.js (hunk 1 above)
Comment (alisonjoseph) [reply src/components/ToggleSmall/ToggleSmall.js]: Should we be adding a comment for every property moving forward, or just the properties we think need an explanation?

File: src/components/ToggleSmall/ToggleSmall.js (hunk 2 above)
Comment (asudoh) [reply src/components/ToggleSmall/ToggleSmall.js]: Yes @alisonjoseph it’s what exactly I meant. Thank you for your quick fix!

File: src/components/ToggleSmall/ToggleSmall.js (hunk 1 above)
Comment (asudoh) [reply src/components/ToggleSmall/ToggleSmall.js]: Good question - I tend to comment everything just for a consistency check, but I see comments for some self-explanatory ones may be useless. My suggestion is being careful with ones that *look* self-explanatory and see if they really are (e.g. if we have `id` where is it applied to?). Thank you for asking!

**Pull request merged**